### ISSUE TITLE
订阅更新功能不能用！

### ISSUE THREAD

**Issue by jojoxl01** [2dust/v2rayNG#108]
手机客户端，安卓4.4版本订阅更新无法更新。

**Comment by mmfan** [comment]
是的!!有兼容性问题!!!部分地址可以用!!订阅地址在浏览器里打开如果是下载个扩展名是txt的文本里面是字符就可以订阅更新!!如果在浏览器里打开后直接显示一串字符就不可以更新!!目前测试是这样

**Comment by jojoxl01** [comment]
多谢指教！也非常感谢你对此软件做出的付出！！！

mmfan <notifications@github.com> 于2018年9月2日周日 下午2:39写道：

**Comment by jojoxl01** [comment]
望能够解决此问题！

jo jo <jojo042625@gmail.com> 于2018年9月6日周四 上午10:30写道：
